src/components/Select/index.tsx

```diff
@@ -14,6 +13,18 @@ type AdjustedGrommetSelectProps = Omit<
 	'onChange' | 'children'
 >;
 
+const getBorderColor = ({
+	theme,
+	invalid,
+}: {
+	theme: any;
```

Comment:
```suggestion
	theme: Theme;
```

Reply:
You can also do this inline (`prop.invalid ? theme.colors.danger.main : 'initial'`)